Error in Google play services

Not entirely sure what happened. I didn't have any google apps open at the time. I was just scrolling through a social media app

I also didn't notice any changes, other than the notification coming up.

```
type: crash
osVersion: google/oriole/oriole:13/TP1A.221105.002/2022110800:user/release-keys
package: com.google.android.gms:224113044
process: com.google.android.gms

dajs: rethrowing exception from lowpool[85]
	at zyb.run(:com.google.android.gms@224113044@22.41.13 (190400-480714934):4)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at arnp.lv(:com.google.android.gms@224113044@22.41.13 (190400-480714934):0)
	at arnp.dispatchMessage(:com.google.android.gms@224113044@22.41.13 (190400-480714934):11)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7904)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)
	Suppressed: cuij: 
		at tk_trace.ScheduledTaskImpl(Unknown Source:0)
		at tk_trace.security__nonwearable-TelecomTaskService_onRunTaskAsync(Unknown Source:0)
		at tk_trace.security__nonwearable-TelecomTaskService-IMessenger_0(Unknown Source:0)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.List android.content.pm.ParceledListSlice.getList()' on a null object reference
	at android.telecom.TelecomManager.getAllPhoneAccountHandles(TelecomManager.java:1472)
	at brui.a(:com.google.android.gms@224113044@22.41.13 (190400-480714934):2)
	at com.google.android.gms.security.snet.TelecomTaskService.a(:com.google.android.gms@224113044@22.41.13 (190400-480714934):2)
	at arpb.call(:com.google.android.gms@224113044@22.41.13 (190400-480714934):4)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at zyr.c(:com.google.android.gms@224113044@22.41.13 (190400-480714934):6)
	at zyr.run(:com.google.android.gms@224113044@22.41.13 (190400-480714934):7)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at aadw.run(:com.google.android.gms@224113044@22.41.13 (190400-480714934):0)
	at java.lang.Thread.run(Thread.java:1012)
```

Just experienced similar crash. Pixel 6a running the correct version of Play Services (directly from Apps, has never been updated via Aurora). I was within the user profile that has sandboxed play installed on it, but not using any apps that required it. Rebooted and everything's working as expected. Reporting just in case it's helpful.

```
type: crash
osVersion: google/bluejay/bluejay:13/TP1A.221105.002/2022110800:user/release-keys
package: com.google.android.gms:224113044
process: com.google.android.gms

dajs: rethrowing exception from lowpool[7]
	at zyb.run(:com.google.android.gms@224113044@22.41.13 (190400-480714934):4)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at arnp.lv(:com.google.android.gms@224113044@22.41.13 (190400-480714934):0)
	at arnp.dispatchMessage(:com.google.android.gms@224113044@22.41.13 (190400-480714934):11)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7904)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ExecInit.main(ExecInit.java:49)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)
	Suppressed: cuij: 
		at tk_trace.ScheduledTaskImpl(Unknown Source:0)
		at tk_trace.security__nonwearable-TelecomTaskService_onRunTaskAsync(Unknown Source:0)
		at tk_trace.security__nonwearable-TelecomTaskService-IMessenger_0(Unknown Source:0)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.List android.content.pm.ParceledListSlice.getList()' on a null object reference
	at android.telecom.TelecomManager.getAllPhoneAccountHandles(TelecomManager.java:1472)
	at brui.a(:com.google.android.gms@224113044@22.41.13 (190400-480714934):2)
	at com.google.android.gms.security.snet.TelecomTaskService.a(:com.google.android.gms@224113044@22.41.13 (190400-480714934):2)
	at arpb.call(:com.google.android.gms@224113044@22.41.13 (190400-480714934):4)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at zyr.c(:com.google.android.gms@224113044@22.41.13 (190400-480714934):6)
	at zyr.run(:com.google.android.gms@224113044@22.41.13 (190400-480714934):7)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at aadw.run(:com.google.android.gms@224113044@22.41.13 (190400-480714934):0)
	at java.lang.Thread.run(Thread.java:1012)

```

--- a/config-holder/app/res/values/strings.xml
+++ b/config-holder/app/res/values/strings.xml
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">GmsCompat config</string>
+    <string name="app_name">GmsCompatConfig</string>
 </resources>

--- a/gmscompat_config
+++ b/gmscompat_config
@@ -40,6 +40,10 @@ ReceiverControlChannelFeatures__enable_user_accept_dialogs false
 # controls access to ContactsContract$SimContacts#getSimAccounts()
 AccountCategoriesFeature__use_cp2_sim_account_settings perms-none-of android.permission.READ_CONTACTS false
 
+[com.google.android.gms.security]
+# disable TelecomTaskService, it needs privileged MODIFY_PHONE_STATE permission
+enable_telecom_feature false
+
 [com.google.android.gms.update]
 # OS update service
 service_enabled false

